AlphaWallet/AppCoordinator.swift

```diff
@@ -175,8 +205,25 @@ class AppCoordinator: NSObject, Coordinator {
         if let coordinator = inCoordinator {
             return coordinator
         } else {
-            return showTransactions(for: keystore.currentWallet)
+            return showTransactions(wallet: keystore.currentWallet, animated: false)
+        }
+    }
+
+    //NOTE: Not sure how to call it right, rename with more appropriate name.
+    private enum AccountsStartPoint {
+        case withAccountsPresentation
+        case skipAccounts
+    }
+
+    @discardableResult private func showTransactions(wallet: Wallet, animated: Bool, some: AccountsStartPoint = .withAccountsPresentation) -> InCoordinator {
+        switch some {
```

Comment:
Can we make this not reload the UI if the current account is tapped? Something like https://github.com/AlphaWallet/alpha-wallet-ios/blob/f8f132f5ecfb7cfde5d640996d40c9da3ad5d2c4/AlphaWallet/Settings/Coordinators/SettingsCoordinator.swift#L223